scripts/common.py

```diff
@@ -437,6 +437,19 @@ def convert_to_posixpath(file_path):
     return file_path.replace('\\', '/')
 
 
+def create_readme(dir_path, readme_content):
+    """Creates a readme in a given dir path with the specified
+    readme content.
+
+    Args:
+        dir_path: str. The path of the dir where the README is to
+            be created.
+        readme_content: str. The content to be written in the README.
+    """
+    with python_utils.open_file('%s/README.md' % dir_path, 'w') as f:
```

Comment:
Use os.path.join for filepaths, for platform independence.

Reply:
Done.